authentication/typescript/azure/src/azureIdentityAuthenticationProvider.ts

```diff
@@ -1,21 +1,15 @@
-import { TokenCredential } from "@azure/core-auth";
-import { BaseBearerTokenAuthenticationProvider, RequestInformation  } from '@microsoft/kiota-abstractions';
+import { GetTokenOptions, TokenCredential } from "@azure/core-auth";
+import { BaseBearerTokenAuthenticationProvider } from '@microsoft/kiota-abstractions';
+import { AzureIdentityAccessTokenProvider } from "./azureIdentityAccessTokenProvider";
 
 export class AzureIdentityAuthenticationProvider extends BaseBearerTokenAuthenticationProvider {
```

Comment:
why do we need this class at all, since you can new a basebearer provider and new up the Azure token provider.

Reply:
This is just sugar for the consumers so they don't have to instantiate two different objects

Reply:
i can live with that.